packages/open-payments/src/client/grant.ts

```diff
@@ -17,10 +18,18 @@ interface RequestGrantArgs {
   request: Omit<GrantRequest, 'client'>
 }
 
+interface ContinuationRequestArgs {
+  url: string
+  request: Omit<ContinuationRequest, 'client'>
+}
+
 export interface GrantRoutes {
   request(
     args: RequestGrantArgs
   ): Promise<InteractiveGrant | NonInteractiveGrant>
+  continue(
+    args: ContinuationRequestArgs
+  ): Promise<InteractiveGrant | NonInteractiveGrant>
```

Comment:
```suggestion
  continue(
    args: ContinuationRequestArgs
  ): Promise<NonInteractiveGrant>
```
I believe we will get back a `NonInteractiveGrant` only that will contain the `continue` and the `access_token` fields